tests/integration_tests/test_lm/test_jsrun.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=protected-access, unused-argument, no-value-for-parameter
+import os
+import subprocess as sp
+
+import radical.utils as ru
+
+from unittest import mock, TestCase
+
+from radical.pilot.agent.launch_method.jsrun import JSRUN
+
+
+# ------------------------------------------------------------------------------
+#
+class TestTask(TestCase):
+
+    # --------------------------------------------------------------------------
+    #
+    @classmethod
+    def setUpClass(cls):
+        path = os.path.dirname(__file__) + '/../test_config/resources.json'
+        resources = ru.read_json(path)
+        hostname = 'summit'
+
+        for host in resources.keys():
+            if host in hostname:
+                cls.host = host
+                cls.resource = resources[host]
+                break
```

Comment:
maybe update it as `cls.resource = resources['summit']` (since you know which configuration to pick)

Reply:
I think this is fixed.